docs/api/modules/App-Configuration.md

```diff
@@ -521,7 +523,38 @@ The `dnsCache` option allows for enabling application-level DNS caching. It is d
 
 `maxTtl` affects the maximum lifetime of the entries received from the specified DNS server (TTL in seconds). If set to 0, it will make a new DNS query each time. It is not recommended to be lower than the DNS server response time in order to prevent bottlenecks.
 
+## `redirectAllowList`
 
+**Module Type**
+
+- ✅ Root Module
+- 🚫 Child Module
+
+**Shape**
+
+```js
+if (!global.BROWSER) {
+  Module.appConfig = {
+    redirectAllowList: ['string'],
+  };
+}
+```
+
+The `redirectAllowList` config option allows you to configure a list of domains that the server is allowed to redirect to. Each URL in this list should include the protocol (`https://`).
```

Comment:
This could use a little context as to what would cause an external redirect.

Reply:
This is specifically for this:
https://github.com/americanexpress/one-app/blob/main/docs/api/modules/Loading-Data.md#holocron-module-configuration

Do you think adding a link to that portion of the docs is enough?

Reply:
Yeah, that would add some context.